spec/CookiePluginSpec.php

```diff
@@ -146,7 +146,12 @@ function it_saves_cookie(RequestInterface $request, ResponseInterface $response,
 
         $response->hasHeader('Set-Cookie')->willReturn(true);
         $response->getHeader('Set-Cookie')->willReturn([
-            'cookie=value',
+            sprintf(
+                'cookie=value; expires=%s; Max-Age=60; path=/; domain=test.com; secure; HttpOnly',
+                (new \DateTime())->add(
+                    new \DateInterval('PT60S')
```

Comment:
Symfony provides a way to safely test time related code in the PHPUnit bridge: http://symfony.com/doc/current/components/phpunit_bridge.html#time-sensitive-tests

Maybe this of some help.